File: trunk/dev/iOS/BomberKlob/BomberKlob/Map.m
#import "Map.h"


@implementation Map

@synthesize name, width, height, players;

- (id)init {
    self = [super init];
    
    if (self) {
        
    }
    
    return self;
}


- (void)dealloc {
    [name release];
    [players release];
    [super dealloc];
}

@end

File: dev/Android/src/com/klob/Bomberklob/resources/Map.java
package com.klob.Bomberklob.resources;

import java.io.Serializable;

public abstract class Map implements Serializable {

	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;

	protected int width, height;

	protected String name;
	protected Point[] players;
	
	/* Constructeur -------------------------------------------------------- */
	public Map () {
		this.players = new Point[4];
	}
	
	/* Getters ------------------------------------------------------------- */
	
	public String getName() {
		return this.name;
	}
	
	public Point[] getPlayers() {
		return players;
	}
	
	public int getWidth() {
		return width;
	}

	public int getHeight() {
		return height;
	}
	
	/* Setters ------------------------------------------------------------- */
	
	public void setName(String name) {
		this.name = name;
	}

}
